packages/PriceLockup/PriceLockup.jsx

```diff
@@ -91,52 +102,108 @@ const StyledWrapperAlignment = styled(Box)({
-        </StyledLargeDollarSign>
-      )
-    }
+const StyledFootnoteLinks = styled(StyledText)(({ inline }) => ({
+  display: 'inline-block',
+  alignSelf: 'flex-start',
+  marginTop: !inline && '0.5rem',
+}))
+
+const StyledBottomText = styled(StyledText)({ display: 'inline-block' })
+
+const renderDollarSign = size => {
+  if (size === 'large') {
     return (
-      <StyledDollarSign data-testid="dollarSign" size={size}>
+      <StyledLargeDollarSign data-testid="dollarSign" as="span" level="h1">
         &#36;
-      </StyledDollarSign>
+      </StyledLargeDollarSign>
     )
   }
+  return (
+    <StyledDollarSign data-testid="dollarSign" size={size}>
+      &#36;
+    </StyledDollarSign>
+  )
+}
 
-  const renderPriceValueSign = () => {
+const renderFootnoteLinks = (footnoteLinksRef, footnoteLinks, inline) => (
+  <StyledFootnoteLinks ref={footnoteLinksRef} inline={inline}>
+    {footnoteLinks}
+  </StyledFootnoteLinks>
+)
+
+const renderBottomText = (size, bottomText, bottomTextRef) => {
+  if (size !== 'large' && bottomText) {
     return (
-      <Box between={size === 'large' ? 2 : 1} inline>
-        {signDirection === 'left' ? renderDollarSign() : undefined}
-        <StyledPriceValue data-testid="priceValue" size={size}>
-          {price}
-        </StyledPriceValue>
-        {signDirection === 'right' ? renderDollarSign() : undefined}
-      </Box>
+      <StyledBottomText ref={bottomTextRef} size={size}>
+        {bottomText}
+      </StyledBottomText>
     )
   }
+  if (size === 'large' && bottomText) {
+    warn('PriceLockup', "The props bottomText and size='large' cannot be used together")
+    return undefined
+  }
+  return undefined
+}
 
-  const renderBottomText = () => {
-    if (size !== 'large' && bottomText) {
-      return <Text size={size}>{bottomText}</Text>
-    }
-    if (size === 'large' && bottomText) {
-      warn('PriceLockup', "The props bottomText and size='large' cannot be used together")
-      return undefined
+/**
+ * A component presenting TELUS product pricing information.
+ * @version ./package.json
+ */
+
+const PriceLockup = ({
+  size,
+  price,
+  topText,
+  signDirection,
+  rateText,
+  bottomText,
+  footnoteLinks,
+}) => {
+  const rateTextWrapperRef = useRef()
+  const footnoteLinksRef = useRef()
+  const containerRef = useRef()
+  const bottomTextRef = useRef()
+
+  const [footnoteLinksInline, setFootnoteLinksInline] = useState(true)
+
+  const checkInline = () => {
+    if (containerRef.current && footnoteLinksRef.current) {
+      const footnoteLinksWidth = footnoteLinksRef.current.offsetWidth
+      const containerWidth = containerRef.current.offsetWidth
+
+      let textWidth
+      if (bottomText) {
+        textWidth = bottomTextRef.current ? bottomTextRef.current.offsetWidth : 0
+      } else {
+        textWidth = rateTextWrapperRef.current.offsetWidth
+      }
+
+      const combinedWidth = textWidth + footnoteLinksWidth
+
+      if (combinedWidth < containerWidth) {
```

Comment:
The `if` and `else` statements can be removed. Then, this can be simplified to a one liner: `setFootnoteLinksInline(combinedWidth < containerWidth)`